.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-    "printWidth": 140,
+    "printWidth": 120,
```

Comment:
Was it agreed between all team members?

Reply:
No,  it was not any discussion about that. I used default recommendation from documentation 
`In code styleguides, maximum line length rules are often set to 100 or 120. `

[link](https://prettier.io/docs/en/rationale.html#print-width)